### PR TITLE
fix some typos, improve some language

### PR DESCRIPTION
**Pull request by martinlindhe**

### CODE REVIEW

File: diesel_tests/tests/bench.rs [hunk 1]
```diff
@@ -163,7 +163,7 @@ bench_medium_complex_query!(
 
 #[bench]
 fn loading_associations_sequentially(b: &mut Bencher) {
-    // SETUP A FUCK TON OF DATA
+    // SETUP A TON OF DATA
```
Comment (martinlindhe) [inline diesel_tests/tests/bench.rs]: I felt the wording here should be changed after reading your https://github.com/diesel-rs/diesel/blob/master/code_of_conduct.md

File: diesel_cli/src/database.rs [hunk 2]
```diff
@@ -255,7 +255,7 @@ table! {
 fn pg_database_exists(conn: &PgConnection, database_name: &str) -> QueryResult<bool> {
     use self::pg_database::dsl::*;
 
-    pg_database.select(datname) // here come dat name!!!! o shit waddup!!!!
+    pg_database.select(datname)
```
Comment (martinlindhe) [inline diesel_cli/src/database.rs]: Not sure what this comment brings to the table, so I removed it.

**Comment by martinlindhe** [comment]
The CI failures seems to be unrelated to this PR, as master is failing on the same issue (https://travis-ci.org/diesel-rs/diesel/builds/324571562)

```
error: `Queryable` is ambiguous
 --> tests/queryable.rs:9:49
  |
9 |     #[derive(Debug, Clone, Copy, PartialEq, Eq, Queryable)]
  |                                                 ^^^^^^^^^
  |
```

File: diesel_cli/src/database.rs (hunk 2 above)
Comment (killercup) [reply diesel_cli/src/database.rs]: Was probably one of those 2017 jokes that no-one laughs about anymore.

But hey, it was introduced  #666!

File: examples/postgres/advanced-blog-cli/src/cli.rs [hunk 3]
```diff
@@ -16,7 +16,7 @@ pub enum Cli {
     EditPost {
         /// The id of the post to edit
         post_id: i32,
-        /// Announce this piece of literary perfectionims to the world?
+        /// Announce this piece of literary perfectionisms to the world?
```
Comment (killercup) [inline examples/postgres/advanced-blog-cli/src/cli.rs]: Very good change. Makes it even more perfect.

**Comment by killercup** [comment]
Sorry, merging #1367 introduced a conflict here. Can you rebase on current master?

(#1367 actually rewrote most of the guide, so maybe you can even find new typos? :D)

**Comment by martinlindhe** [comment]
Sure, I will update this PR

**Comment by martinlindhe** [comment]
PR rebased, and the new guide included too.

File: diesel_cli/src/database.rs (hunk 2 above)
Comment (sgrif) [reply diesel_cli/src/database.rs]: Drunk Sean thought it was funny

**Pull request merged**